component/src/main/java/org/wso2/extension/siddhi/store/rdbms/RDBMSEventTable.java

```diff
@@ -1515,7 +1515,14 @@ private void createTable(Annotation storeAnnotation, Annotation primaryKeys, Ann
                 log.debug("Table '" + this.tableName + "' created.");
             }
         } catch (RDBMSTableException e) {
-            throw new RDBMSTableException("Unable to initialize table '" + this.tableName + "'", e);
+            if (e.getCause().getMessage().contains("already exists")) {
```

Comment:
Possible NPE at `e.getCause()`. If only the message is passed at the `RDBMSTableException` creation, the cause can be null (nonexistent or unknown) and that can lead to an NPE.